src/components/RecentEventsCard.astro

```diff
@@ -21,7 +35,7 @@ const { title, pictureUrl, date, description } = event;
       {title}
     </h1>
     <p class="text-sm md:text-[16px] text-gray-500 mb-3">
-      {date}
+      {getDate()}
     </p>
     <p class="text-sm md:text-lg font-montserrat line-clamp-2">{description}</p>
```

Comment:
We need to use the `tiptap` library to properly display the rich content in the description. 

See sample implementation here:
https://github.com/SPARCS-UP-Mindanao/TechTix/blob/ced512e6b28701ea0306eba4a53c308e16c6039c/frontend/src/components/RichContent/RichTextContent.tsx